sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.NUnit.cs

```diff
@@ -3,6 +3,7 @@
     using System;
     using System.Collections.Generic;
     using FluentAssertions;
+    using NSubstitute;
```

Comment:
it would make sense to add a simple test for `xunit` or other libraries that can be used together with `nsubstitute`, something like 

```
        [TestCase]
        public void ReceivedNameSpace() => NSubstitute.SubstituteExtensions.Received(calculator).Add(1, 2);

```
in each, just to make sure that we don't raise on other libraries then nunit

and then this file will still have extensive tests for nsubstitute specifically (and you can add a comment in the xunit tests that the extensive tests are in this file - so that people know to modify only these in case of a change of nsubstitute API which will need more extensive tests